src/sage/stats/distributions/discrete_gaussian_lattice.py

```diff
@@ -185,28 +187,29 @@ def compute_precision(precision, sigma):
         precision = max(53, precision)
         return precision
 
-    def _normalisation_factor_zz(self, tau=3):
+    def _normalisation_factor_zz(self, tau=None, prec=None):
         r"""
-        This function returns an approximation of `∑_{x ∈ \ZZ^n}
-        \exp(-|x|_2^2/(2σ²))`, i.e. the normalisation factor such that the sum
-        over all probabilities is 1 for `\ZZⁿ`.
+        This function returns an approximation of `\Sigma_{x \in B}
+        \exp(-|x|_2^2 / (2\sigma²))`, i.e. the normalisation factor such that the sum
```

Comment:
```suggestion
        This function returns an approximation of `\sum_{x \in B}
        \exp(-|x|_2^2 / (2\sigma^2))`, i.e. the normalization factor such that the sum
```
We also typically use American English spellings (but you don't have to go and fix this everywhere; I just noticed it here).

Reply:
Oh no 😂 What should the method be named then? If the American spelling is preferred *and* this is an internal method, might as well change it right?

Reply:
Internal methods can be changed without any deprecation (which is only needed for anything public). Feel free to change it or decide to keep it as-is. (For reference, I am American, so I am biased in this. :P )